src/main/java/io/supertokens/webserver/api/core/DeleteUserAPI.java

```diff
@@ -0,0 +1,59 @@
+/*
+ *    Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ *    This software is licensed under the Apache License, Version 2.0 (the
+ *    "License") as published by the Apache Software Foundation.
+ *
+ *    You may not use this file except in compliance with the License. You may
+ *    obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *    License for the specific language governing permissions and limitations
+ *    under the License.
+ */
+
+package io.supertokens.webserver.api.core;
+
+import com.google.gson.JsonObject;
+import io.supertokens.Main;
+import io.supertokens.authRecipe.AuthRecipe;
+import io.supertokens.pluginInterface.exceptions.StorageQueryException;
+import io.supertokens.webserver.InputParser;
+import io.supertokens.webserver.WebserverAPI;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class DeleteUserAPI extends WebserverAPI {
+    public DeleteUserAPI(Main main) {
+        super(main, "");
+    }
+
+    @Override
+    public String getPath() {
+        return "/users/delete";
+    }
+
+    @Override
+    protected void doDelete(HttpServletRequest req, HttpServletResponse resp) throws IOException, ServletException {
+        JsonObject body = InputParser.parseJsonObjectOrThrowError(req);
```

Comment:
Here you are expecting a JSON body - however in the API, you had added the userId as a query param.. so please either change the API spec or change this to read from query params.

Reply:
Or perhaps we should make this API `/user/remove POST` like how we have done for `/recipe/session/remove POST`.